pom.xml

```diff
@@ -6,7 +6,7 @@
 
     <groupId>com.github.kmehrunes</groupId>
     <artifactId>javalin-jwt</artifactId>
-    <version>0.3</version>
+    <version>0.4-SNAPSHOT</version>
```

Comment:
We don't have multiple concurrent changes, and this will go into a full version right away. You can make it 0.4 instead.

Reply:
I will change it at the end while I rebase the branch to clean it up if your ok with it?